test/memory_coloring_test.cpp

```diff
@@ -680,6 +714,3046 @@ TEST_CASE(test39)
     CHECK(no_allocate(*else_mod));
 }
 
+TEST_CASE(rnn_dom)
+{
```

Comment:
should we move this to another file or generate the module in a way to avoid the tidy complexity warning?

Reply:
I dont think there is, I can just add a `NOLINT` to the function.